stdlib/slack/slack.flux

```diff
@@ -126,7 +118,7 @@ endpoint = (url=defaultURL, token="") => (mapFn) => (tables=<-) => tables
     |> map(
         fn: (r) => {
             obj = mapFn(r: r)
-
+    
```

Comment:
This is a small thing, but if we can avoid indenting blank lines that would be good.